documentation/docs/client/channel_members.md

```diff
@@ -8,9 +8,9 @@ sidebar_position: 10
 
 Sometimes channels will have many hundreds (or thousands) of members and it is important to be able to access ID's and information on all of these members. The `queryMembers` endpoint queries the channel members and allows the user to paginate through a full list of users in channels with very large member counts. The endpoint supports filtering on numerous criteria to efficiently return member information.
 
-> The members are sorted by created_at in ascending order.
+> The members are sorted by _created_at_ in ascending order.
 
-> Stream Chat does not run MongoDB on the backend, only a subset of the query options are available.
+> _Stream Chat_ does not run MongoDB on the backend, only a subset of the query options are available.
```

Comment:
Anything else we want to italicize?

Reply:
Good question. I just saw that `Stream Chat` was italicized in some places but not everywhere so I decided to make it consistent.
 IMO we should italicize things like event types names, flags (true/false), etc.